Project.toml

```
@@ -1,7 +1,7 @@
 name = "RootedTrees"
 uuid = "47965b36-3f3e-11e9-0dcf-4570dfd42a8c"
 authors = ["Hendrik Ranocha <mail@ranocha.de> and contributors"]
-version = "2.20.0"
+version = "3.0.0"
```

```suggestion
version = "2.21.0"
```
It's not a breaking change to deprecate something. Deprecating the old function means that it's still accessible. It just informs people that they should use another function in the future (if deprecation warnings are turned on).